controller/src/main/java/org/jboss/as/controller/OperationDefinition.java

```diff
@@ -64,6 +65,7 @@ protected OperationDefinition(String name,
                                final DeprecationData deprecationData,
                                AttributeDefinition[] replyParameters,
                                AttributeDefinition[] parameters,
+                               final DescriptionProvider descriptionProvider,
                                AccessConstraintDefinition... accessConstraints
```

Comment:
This and the subclass constructors are public API, so I prefer not to change their signatures. I'd rather deprecate them and move to passing the builder as the param. See #1009.